### PR TITLE
Update Debian Install Script To Support Debian 10

### LINKED ISSUE #11539: Debian Install Script Not Updated To Support Debian 10
# Steps to reproduce

From a Debian 10 machine:
1. execute the following command
`wget https://aka.ms/install-powershell.sh; sudo bash install-powershell.sh -preview; rm install-powershell.sh`

2. See that installation will fail on execution of the following script in this project repo:
`tools/installpsh-debian.sh`

[Line 189](https://github.com/PowerShell/PowerShell/blob/master/tools/installpsh-debian.sh#L189) has not been updated to reflect the new support for Debian 10.

# Expected behavior

Install script should execute and install PowerShell on Debian 10

# Actual behavior

Script fails with the following error message
```
(23) Failed writing body
ERROR: unsupported Debian version (10).
Supported versions: 8, 9.
For additional versions open an issue or pull request at: https://github.com/powershell/powershell
```
# Environment data

Distro: Debian 10.

I have a fix tested and ready to go. Creating this issue to have something to reference in the Pull Request.

### PR DESCRIPTION
**Pull request by RandomNoun7**
# PR Summary

Support for installing pwsh on Debian 10 was added in #11236, but the
install script for Debian was not updated. This resulted in an error
when running the install script that claimed the distro was not
supported.

This change simply adds `10` to the collection of supported distro's.

I have tested that the script will now successfully install powershell
preview 7 on a Debian 10 machine.

Fix #11539

## PR Checklist

- [ ] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [ ] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [ ] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [ ] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [ ] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [ ] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary. This may include:
        - Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode (which runs in a different PS Host).
        - Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
        - Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
        - Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).

### CODE REVIEW

File: tools/installpsh-debian.sh [hunk 1]
```diff
@@ -186,7 +186,7 @@ case $DISTRIB_ID in
     debian)
         DISTRIB_RELEASE=${DISTRIB_RELEASE%%.*}
         case $DISTRIB_RELEASE in
-            8|9)
+            8|9|10)
```
Comment (TravisEz13) [inline tools/installpsh-debian.sh]:
```suggestion
            8|9|10|11)
```

Comment (RandomNoun7) [reply tools/installpsh-debian.sh]: Done. Thanks.

**Comment by TravisEz13** [comment]
@PoshChan Please remind me in 1 hour

**Comment by PoshChan** [comment]
@TravisEz13, this is the reminder you requested 1 hour ago

**Review by TravisEz13 — APPROVED** [review]

**Pull request merged**